### DISCUSSION TITLE
国外节点http报错连接映射目标失败

### DISCUSSION THREAD

**Discussion by tcitr-chen** [natfrp/wiki#186] [使用问题]
![图片](https://user-images.githubusercontent.com/61141667/195975579-4b6f1ac7-1739-4b1d-9379-9d396d25358e.png)
国内节点访问正常
![图片](https://user-images.githubusercontent.com/61141667/195975603-4c152c13-2d47-4f78-9513-0a6e80da0e72.png)
国外节点报错如上
面板和ping如下
![图片](https://user-images.githubusercontent.com/61141667/195975816-17a40e23-b850-47fc-91a2-c133cc25e9d8.png)
![图片](https://user-images.githubusercontent.com/61141667/195975725-65ffe2bb-6fc0-4cf7-acad-ec04392cc28a.png)

**Comment by fakeboboliu** [comment]
连接映射目标失败，请检查本地服务是否打开： http://192.168.3.6

**Reply by tcitr-chen** [reply]
本地服务已经打开，我内网试了可以打开，换国内节点也正常，这两个海外节点就是不行。
